### PR TITLE
api: task_manager: rename id to task_id in task_status

### PR DESCRIPTION
**Pull request by Deexie**
No backport needed

### CODE REVIEW

**Comment by Deexie** [comment]
cc @yarongilor

**Comment by denesb** [comment]
Why? (please add it to the patch description)

File: api/api-doc/task_manager.json [hunk 1]
```diff
@@ -291,7 +291,7 @@
          "id":"task_status",
          "description":"A task status object",
          "properties":{
-            "id":{
+            "task_id":{
                "type":"string",
```
Comment (avikivity) [inline api/api-doc/task_manager.json:295]: Isn't this a public API? What about backward compatibility?

Comment (Deexie) [reply api/api-doc/task_manager.json:295]: Yes, it is. I also planned on dropping the fields that are unused by the task (like not showing keyspace if it isn't specified), but I guess I need to give up on both

**Comment by Deexie** [comment]
Closing because of the above

**Pull request closed without merging**